R/module_teal.R

```diff
@@ -42,42 +42,8 @@ NULL
 
 #' @rdname module_teal
 #' @export
-ui_teal <- function(id,
-                    modules,
-                    title = build_app_title(),
-                    header = tags$p(),
-                    footer = tags$p()) {
+ui_teal <- function(id, modules) {
```

Comment:
documentation of `module_teal` needs to be modified as it says `this module can be called directly instead of and included in your custom application`. We need to note that it lacks header and footer.

Reply:
Please have a look at the document now.

Reply:
I think there is a duplicated content between @description and @details section. I'd shorten the description and extend details (without repeating same meaning in few sentences).

The description - `Module to create a `teal` app as a Shiny Module`
The details - the rest combined.

P.S. `reporter_previever_module` should probably be inserted by `ui/srv_teal`. I'll think about this (not for this PR)